tests/bson/bson-binary-jsonserialize-001.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+MongoDB\BSON\Binary::jsonSerialize() return value
```

Comment:
These "return value" tests are new. The "json_encode()" tests are based on your own (with different code formatting and `===DONE===` blocks.